torch/csrc/distributed/c10d/symm_mem/CUDASymmetricMemoryOps.cu

```diff
@@ -152,7 +152,7 @@ static __global__ void multimem_all_reduce_kernel(
   }
 
   __syncthreads();
-  sync_remote_blocks<std::memory_order_acq_rel>(signal_pads, rank, world_size);
+  sync_remote_blocks<true, true>(signal_pads, rank, world_size);
```

Comment:
So, if we have written to remote ranks' user buffer, we'd need to use <true, true>, because the next kernel (on remote rank) can be any kernel which may not perform an acquire, correct?